packages/@ourworldindata/grapher/src/mapCharts/GlobeController.ts

```diff
@@ -172,3 +209,203 @@ export class GlobeController {
             })
     }
 }
+
+function calculateTargetForCountry(country: EntityName): Target | undefined {
+    const geoFeature = geoFeaturesById.get(country)
+    if (!geoFeature) return
+
+    const { centroid } = geoFeature
+    const coords: [number, number] = [-centroid[0], -centroid[1]]
```

Comment:
Just to understand this: how come we're using negative coordinates always?

If it's easy to change, it might be clearer to do that?

Reply:
I'm also not so sure to be honest. I guess we could do the inverse earlier (when we compute the geo features), so we don't have to deal with that complexity later

Reply:
Yeah, I think that'll be useful: do it as early as possible and where it still makes sense.